src/@utils/subgraph.ts

```diff
@@ -238,6 +250,26 @@ export async function fetchDataForMultipleChains(
   return datas
 }
 
+export async function getOpcFeees() {
+  let opcFees
+  const variables = {
+    id: 1
+  }
+  const context = getQueryContext(4)
```

Comment:
what's the hardcoded `chainId` doing there? In publish form, need to switch that dynamically based on user wallet

Reply:
Right, I realised after making it ready for review and changed it

Reply:
haha yes, you pushed the change when I hit submit on this comment, great timing :-)